client/src/layout/Footer/Footer.module.scss

```diff
@@ -0,0 +1,126 @@
+.container {
```

Comment:
write the styles to the footer.js by sx (so that there is no intersection with the styles material and styles.module.sss)

Reply:
Ready, now all styles were made using SX and Styled components. Please, review